### ISSUE TITLE
Cheaper rune/arcane recipes

### ISSUE THREAD

**Issue by MarioSMB** [iron431/Irons-Spells-n-Spellbooks#258]
### Feature description

While it is certainly easy to gather large amounts of the resources needed to craft runes and such, I believe it would provide a more fair experience and respectful to one's time to only require 4 focus items surrounding blank runes to craft the elemental runes - likewise for arcane cloth and ingots.  
Blank runes are rare enough as is; I would also suggest allowing crafting a rune with a bucket of water to "clean" it and obtain a blank rune for reuse.

### How it improves the player experience

As mentioned in the description, it would be more respectful to the player's time to require only 4 items per rune and essence per cloth without subtracting from the experience.

**Comment by lab3** [comment]
You can already modify recipes, add recipes and change spawn rates with a data pack if you would like to change the balance of the game.

**Comment by MarioSMB** [comment]
I have already changed it on my server and we found that it made for a much better experience, hence my putting forth the suggestion here for more sensible defaults.  

EDIT: I implore you to genuinely consider this suggestion; it is easy to brush off such requests as a "do it yourself" type situation - yes, users can adjust values, thank you for that ability, but defaults are not set in stone and they can shape the first impression of the mod.  
There is no good reason to require so much investment in a single item when you need multiple - for example you need an entire stack of phantom membranes to make a single recovery upgrade orb, which is a ridiculous amount of grinding, especially on a server where players will be skipping the nights.

**Comment by iron431** [comment]
While it is true that a grindy early game could cause a bad first impression, upgrade orbs are very late game and are supposed to be a grind. They offer an extension to progression for players to continue engaging with the content by grinding for the rare magical materials. 
That being said, most materials required for upgrade orbs are farmable, if not automatable. Phantom membrane may be the biggest exception to that, especially if your server plays a certain way, so I would consider that specific grievance to be something you should tailor to your own player base.

**Comment by MarioSMB** [comment]
Apologies for using upgrade orbs as an example - they are indeed late game and something we've just recently stumbled across, not the basis for this suggestion.  
But they do illustrate an issue here (and one that is often reached in game design when looking at ways to extend playtime); that is the added costs to incentivize longer play sessions and grinding to acquire better gear.   

There of course needs to be a balance, as making something too cheap takes away from the experience as players will have "everything" way too quickly, whereas making it too expensive can turn the game into a chore.  
Striking a balance makes the player feel rewarded for the time they've put into creating something.  

A great example would be the spellbooks, specifically the Ancient Codex; it was actually fun gathering a reasonable amount of resources (4 debris in particular) to be able to assemble it, the progression was near-perfect. Not to mention a trip to the ancient cities for the ruined book, a good bit of exploration without requiring excessive amounts of each material.   
On the other hand, assembling a full set of armor (requiring 192 arcane essence and 4 blank runes which can be hard to come across) was a chore, and I wouldn't even consider making a second set to try out speccing for other elements.  

For this reason I believe making the recipe just that bit cheaper would help a lot with the experience of progressing in this mod.

**Comment by iron431** [comment]
closing because feature requests are now exclusively being handled on discord https://discord.gg/TRzEdrndM2

**Comment by MarioSMB** [comment]
Needing to farm 32 poisonous potatoes to make a full set of nature armour is one of many grinds that don't seem worthwhile, after almost a year I'd hope this issue was automatically added to any new list of suggestions.

**Comment by iron431** [comment]
if we're not handling your personal gripes quickly enough, feel free to write a design doc for an economy overhaul and ill take a look :)

**Comment by MarioSMB** [comment]
Given this issue has been closed twice and now it’s being called a personal gripe, I do not see it worth the investment to further push this as there does not appear to be any interest in rebalancing the amounts of materials required to craft the core items.

**Comment by lab3** [comment]
Legit concerns get meaningful discussion on discord and are likely to get acted upon. But I think personal gripe is fair characterization given we have 15M downloads and 1 complaint about poisonous potatoes.

**Comment by MarioSMB** [comment]
From that statement it is clear that I would continue to be personally targeted for presenting this issue to the team if I joined the Discord.  
Just because only one person takes the time to report an issue, that doesn’t mean they’re the only one with the issue…

But if what you say is true and nobody else cares about the resource costs for crafting everything, then by all means we can let this issue go.

**Comment by absolute-val** [comment]
Another complaint from me.
I have runned 50km in my present level, only to find 24 runes(from every clerics,fire mages,hidden chests I found from domestic wiki information), And I do need to find another 96 rune stones for my final equipment. The time cost makes no sense despite the final game phase.
By the way,75% extra spell power might be too high. Sincerely hope the author could balance both time cost and strength, instead of making player invulnerable after effort of about one month.
